docs/content/preview/faq/yugabyte-platform.md

```diff
@@ -123,3 +138,88 @@ Yes, you can control what YugabyteDB Anywhere is spinning up. For example:
 - You can choose dedicated IOPs EBS drives on AWS and specify the number of dedicated IOPS you need.
 
 YugabyteDB Anywhere also allows creating these machines out of band and importing them as an on-premises install.
+
+## Node agent
+
+### What is a node agent?
+
+Node agent is an RPC service running on a YugabyteDB node, and is used to manage communication between YugabyteDB Anywhere and the database node. It includes the following functions:
+
+- Invoke shell commands directly on the remote host like running commands over SSH. Similar to SSH, the agent also does shell-login such that any previous command modifying the environment in the resource files (for example, `~/.bashrc`) gets reflected in the subsequent command.
+- Invoke procedures or methods on the node agent.
+- For on-premesis (manual provisioned nodes), node agent functions as a utility to run preflight checks, and add a node instance.
+
+### How is node agent installed on a YugabyteDB node?
+
+Installation depends on how the nodes are provisioned. For cloud (AWS, GCP, and Azure) and non-manual on-prem nodes (sudo access provided), the agents are automatically installed as a part of a universe task during VM provisioning using SSH. So, a sudo user with SSH access (for example, ec2-user for AWS) is needed. After the node agent is installed, all the legacy SSH calls are replaced with node agent calls. Just like SSH daemon, node agent is run as a root user to perform provisioning of the nodes. After node provisioning, you can revoke the SSH access, but it's recommended to enable the access for debugging.
+
+For on-prem manual provisioning, an SSH user does not have sudo access, but sudo is available on the local YugabyteDB node with a password.
+
+For more information, refer to [install node agent](../../yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual/#install-node-agent).
+
+### Why does YBA UI ask for SSH if the node agent is installed manually and can replace SSH?
+
+This is because of the legacy UI. There are some cases where an SSH key is needed for integrity but a fictitious non-working SSH key can be supplied.
+
+### What is registration and unregistration in node agent?
+
+Node agent is a secure service that performs authentication on every remote call from YBA. The service requires a set-up to make:
+
+- YBA aware of the node agent service.
+- the node agent trust YBA and vice-versa.
+This step is called registration. Note that registration has nothing to do with a provider.
+
+Unregistration is the step to disconnect YBA and a node agent from each other.
```

Comment:
IMO, disconnect can mean aborting an existing connection. May be something like remove node-agent entry from YBA such that they do not know each other.